dspace/config/dspace.cfg

```diff
@@ -1572,6 +1572,23 @@ request.item.type = all
 request.item.helpdesk.override = false
 
 
+#---------------------------------------------------------------#
+#------------SPRING BOOT ACTUATOR CONFIGURATION-----------------#
+#---------------------------------------------------------------#
+
+management.endpoint.health.show-details = when-authorized
+management.endpoint.health.roles = ADMIN
+management.endpoint.health.status.order= down, out-of-service, up-with-issues, up, unknown
+
+management.health.ping.enabled = false
+management.health.diskSpace.enabled = false
+
+management.endpoints.web.cors.allowed-origins = ${rest.cors.allowed-origins}
```

Comment:
Again, this section needs comments to describe that it controls the CORS settings of ALL `/actuator/*` endpoints.  We also should make a note here that you'd want to sync these with the CORS settings in `org.dspace.app.rest.Application` (Sidenote: eventually we should probably just make them all one configuration to avoid duplicating the same thing in both places. But, that can be done in a future PR)

Reply:
Ok I will add a comment